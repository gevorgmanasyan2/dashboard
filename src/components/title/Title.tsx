import React, { FC } from 'react'
import { Typography } from 'antd'

import './title.scss'

interface TitleProps {
  className?: string
  titleText?: string | boolean
  level?: 5 | 1 | 2 | 3 | 4 | undefined
}

const Title: FC<TitleProps> = ({ level, className, titleText }) => {
  const { Title } = Typography
  return (
    <Title level={level} className={`main-title ${className}`}>
      {titleText}
    </Title>
  )
}

export default Title
