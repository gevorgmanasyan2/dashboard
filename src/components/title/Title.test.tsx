/* eslint-disable no-undef */
import React from 'react'
import { shallow } from 'enzyme'
import Title from './Title'

it('should render Title component', () => {
  const mockItem = {
    titleText: 'title',
    className: 'header__title',
  }
  expect(shallow(<Title {...mockItem} />)).toMatchSnapshot()
})
