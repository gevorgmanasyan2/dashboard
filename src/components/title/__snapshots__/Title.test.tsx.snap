// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render Title component 1`] = `
<Title
  className="main-title header__title"
>
  title
</Title>
`;
